src/backend/backend.did

```diff
@@ -29,7 +29,7 @@ type HttpResponse = record {
   headers : vec record { text; text };
   status_code : nat16;
 };
-type IcrcToken = record { ledger_id : principal };
+type IcrcToken = record { ledger_id : principal; index_id : principal };
```

Comment:
This is a breaking change.  If the code on the left has not been deployed, this is fine, otherwise it will break.  How can I tell which code is in prod?

Suggestion:  If release descriptions contained hashes, I could get the prod canister hash and then look for it in the releases page.

Reply:
I also tried to get the git commit from the canister metadata, ut it was not available.  At least not at:
```
dfx canister metadata backend git_commit_id --network ic
```

Reply:
It's definitely not in production. I provided the PR few minutes after we merged the one that introduced the `ledger_id` and discussed Monday, I assumed it was clear it was not in production. Will add a note next time in teh PR description.